apis/fluent/atrium-api-fluent/src/commonMain/kotlin/ch/tutteli/atrium/api/fluent/en_GB/charSequenceExpectations.kt

```diff
@@ -256,3 +256,30 @@ fun <T : CharSequence> Expect<T>.toMatch(pattern: Regex): Expect<T> =
  */
 fun <T : CharSequence> Expect<T>.notToMatch(pattern: Regex): Expect<T> =
     _logicAppend { mismatches(pattern) }
+
+/**
+ * Creates an [Expect] for the property [CharSequence.length] of the subject of `this` expectation,
+ * so that further fluent calls are assertions about it.
+ *
+ * @return The newly created [Expect] for the extracted feature
+ *
+ * @sample ch.tutteli.atrium.api.fluent.en_GB.samples.CharSequenceExpectationSamples.lengthFeature
+ *
+ * @since 1.3.0
+ */
+val <T : CharSequence> Expect<T>.length: Expect<Int>
+    get() = _logic.length().transform()
+
+/**
+ * Expects that the property [CharSequence.length] of the subject of `this` expectation
+ * holds all assertions the given [assertionCreator] creates for it and
+ * returns an [Expect] for the current subject of `this` expectation.
+ *
+ * @return an [Expect] for the subject of `this` expectation.
+ *
+ * @sample ch.tutteli.atrium.api.fluent.en_GB.samples.CharSequenceExpectationSamples.length
+ *
+ * @since 1.3.0
+ */
+fun <T : CharSequence> Expect<T>.length(assertionCreator: Expect<Int>.() -> Unit): Expect<T> =
+    _logic.length().collectAndAppend(assertionCreator)
```

Comment:
something I overlooked during the first review. please move those to a new file called `charSequenceFeatureExtractors.kt`